nilearn/datasets/func.py

```diff
@@ -1669,3 +1669,219 @@ def fetch_cobre(n_subjects=10, data_dir=None, url=None, verbose=1):
 
     return Bunch(func=func, mat_files=mat, phenotypic=csv_array,
                  description=fdescr)
+
+
+def fetch_surf_nki_enhanced(n_subjects=30, data_dir=None,
```

Comment:
May be we need reduce default size in ```n_subjects``` to reduce the burden on CircleCI build ? 10 would be good I guess.